### PR TITLE
GDPR Compliance

### PR DESCRIPTION
**Pull request by khorne3**
https://auth0-docs-content-pr-5068.herokuapp.com/docs/compliance

### CODE REVIEW

File: articles/compliance/definitions.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: General Data Protection Regulation (GDPR) Definitions
+description: Definitions used for Auth0's documentation on GDPR
+---
+# Definitions
+
+| Term | Definition |
+| - | - |
+| Subject | An individual/natural person |
+| Data Controller | The entity that collects and processes data on subjects (see GDPR for exact definition) |
```
Comment (mauricioschneider) [inline articles/compliance/definitions.md]: For convenience, could it be possible to add a link to the official resources where this definition can be found?

File: articles/compliance/definitions.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+| Data Processor | The entity that processes data on behalf of a data controller (see GDPR for exact definition) |
+| Personal Data | Data that can be used to identify (directly or indirectly) a subject, particularly via reference to an identifier (such as a name, identification number, location data, or online identifier), or to the physical, physiological, genetic, mental, economic, cultural, or social identity of that person |
+| Sensitive Personal Data | Personal data that reveals racial or ethnic origin, political opinions, religious or philosophical beliefs, or trade-union membership; genetic data or biometric data |
+| Auth0 Subprocessors | [Third party systems](/compliance/subprocessors) to which Auth0 provides personal data |
```
Comment (mauricioschneider) [inline articles/compliance/definitions.md]: Throwing `404`

File: articles/compliance/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <i class="icon icon-budicon-715"></i><a href="/compliance/data-processing">Auth0 Data Processing</a>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/compliance/features-aiding-compliance">Auth0 Features Aiding Customer Compliance</a>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/compliance/security-advice-for-customers">Security Advice for Customers</a>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/compliance/subprocessors">Auth0 Sub-Processors</a>
```
Comment (mauricioschneider) [inline articles/compliance/index.md]: throwing `404`

File: articles/compliance/gdpr-summary.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+## Requirements for Data Processors and Controllers
+
+As the data controller, you must:
+
+* Do due diligence to ensure that your data processors provide adequate protection of provided data
+
+Auth0, as the data processor, must:
+
+* Comply with instructions provided by data processors
```
Comment (mauricioschneider) [inline articles/compliance/gdpr-summary.md]: Should this be `provided by data controllers` instead?

Comment (yvonnewilson) [reply articles/compliance/gdpr-summary.md]: Yes, good catch - that should have said "Comply with instructions provided by data controllers"

File: articles/compliance/features-aiding-compliance.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+You can use [rules](/rules) to alter privileges or other attributes in the user profile that might help with this obligation.
+
+## Choice of Providers
+
+You can choose which identity providers to use for user authentication.
+
+Using external providers means that your end users' credentials are **not** stored in Auth0 (or onsite).
+
+::: note
+If you use LDAP connections, turn off caching to prevent end user credentials from being store in Auth0/onsite.
```
Comment (nicosabena) [inline articles/compliance/features-aiding-compliance.md]: store -> stored

File: articles/compliance/features-aiding-compliance.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+---
+# Auth0 Features Aiding General Data Protection Regulation (GDPR) Compliance
+
+The following Auth0 features (listed based on the end-goal action of the user or functionality offered to you as the customer) help you comply with GDPR regulations.
+
+## Signup
+
+You can:
+
+* Use [Lock to allow new user signups](/libraries/lock/v10/customization) when using [database connections](/libraries/lock/v10/customization)
```
Comment (mpaktiti) [inline articles/compliance/features-aiding-compliance.md]: Both links point to the same. It's a bit confusing, we should keep only one.

File: articles/compliance/features-aiding-compliance.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+* [Enable or disable signup mode](/libraries/lock/v10/customization#allowsignup-boolean-) in Lock
+* Implement a [custom signup process](/libraries/custom-signup)
+* Implement [progressive profiling](/user-profile/progressive-profiling) to gather only the data you need
+* Implement end-user [multifactor authentication](/multifactor-authentication) to secure access
+
+## Notification and Consent
+
+You can:
+
+* Provide [a **User Must Accept Terms** checkbox](/libraries/lock/v10/customization#mustacceptterms-boolean-) that is displayed next to the terms and conditions the user must agree to prior to signing up with the [verbiage displayed controllable using the language dictionary](/libraries/lock/v10/customization#mustacceptterms-boolean-)
```
Comment (mpaktiti) [inline articles/compliance/features-aiding-compliance.md]: Same, we link twice to the same doc

File: articles/compliance/features-aiding-compliance.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+::: note
+See [signUpTerms](https://github.com/auth0/lock/blob/master/src/i18n/en.js) for a detailed code sample.
+:::
+
+## Withdrawal of Consent
+
+You can:
+
+* [Delete the end user from Auth0 using the Management API](/api/management/v2#!/Users/delete_users_by_id).
+* Use [rules](/rule) to add the date of user consent withdrawal to the user's metadata during the authorization process
```
Comment (mpaktiti) [inline articles/compliance/features-aiding-compliance.md]: `/rule` typo throws 404

File: articles/compliance/index.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
+---
+title: Auth0 and General Data Protection Regulation
+description: How Auth0 complies with the EU's General Data Protection Regulation (GDPR)
+classes: topic-page
+---
+
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>General Data Protection Regulation</h1>
+  <p>
```
Comment (mpaktiti) [inline articles/compliance/index.md]: Add a short text here (within the `topic-page-header` div) and move all this text after the `topic-page-header` div.

Currently, it looks like this:

image

**Review by mpaktiti — CHANGES_REQUESTED** [review]

**Comment by khorne3** [comment]
@mpaktiti Done. Do you have the ability to kick off another Heroku build?

**Pull request merged**